sentry_sdk/integrations/sanic.py

```diff
@@ -276,16 +274,14 @@ async def sentry_wrapped_error_handler(request, exception):
 
 async def _legacy_handle_request(self, request, *args, **kwargs):
     # type: (Any, Request, *Any, **Any) -> Any
-    hub = Hub.current
-    if hub.get_integration(SanicIntegration) is None:
+    if sentry_sdk.get_client().get_integration(SanicIntegration) is None:
         return old_handle_request(self, request, *args, **kwargs)
 
     weak_request = weakref.ref(request)
 
-    with Hub(hub) as hub:
-        with hub.configure_scope() as scope:
-            scope.clear_breadcrumbs()
-            scope.add_event_processor(_make_request_processor(weak_request))
+    with sentry_sdk.isolation_scope() as scope:
```

Comment:
nice! much cleaner

Reply:
Yes! The new API feels much cleaner in many integrations.